vNext/AISKU/snippet/snippet.js

```diff
@@ -1,96 +1,96 @@
-            // Queue-up a call to the real method
-            appInsights.queue.push(function () {
-                // Invoke the real method with the captured original arguments
-                appInsights[name].apply(appInsights, originalArguments);
-            });
-        };
-    }
+        function createLazyMethod(name) {
+            // Define a temporary method that queues-up a the real method call
+            appInsights[name] = function () {
+                // Capture the original arguments passed to the method
+                var originalArguments = arguments;
+                // Queue-up a call to the real method
+                appInsights.queue.push(function () {
+                    // Invoke the real method with the captured original arguments
+                    appInsights[name].apply(appInsights, originalArguments);
+                });
+            };
+        }
 
-    var method = ["Event", "PageView", "Exception", "Trace", "DependencyData", "Metric", "PageViewPerformance"];
-    while (method.length) {
-        createLazyMethod("track" + method.pop());
-    }
+        var method = ["Event", "PageView", "Exception", "Trace", "DependencyData", "Metric", "PageViewPerformance"];
+        while (method.length) {
+            createLazyMethod("track" + method.pop());
+        }
 
-    var track = "Track";
-    var trackPage = track + "Page";
-    createLazyMethod("start" + trackPage);
-    createLazyMethod("stop" + trackPage);
+        var track = "Track";
+        var trackPage = track + "Page";
+        createLazyMethod("start" + trackPage);
+        createLazyMethod("stop" + trackPage);
 
-    var trackEvent = track + method[0];
-    createLazyMethod("start" + trackEvent);
-    createLazyMethod("stop" + trackEvent);
+        var trackEvent = track + method[0];
+        createLazyMethod("start" + trackEvent);
+        createLazyMethod("stop" + trackEvent);
 
-    createLazyMethod("setAuthenticatedUserContext");
-    createLazyMethod("clearAuthenticatedUserContext");
-    createLazyMethod("flush");
+        createLazyMethod("setAuthenticatedUserContext");
+        createLazyMethod("clearAuthenticatedUserContext");
+        createLazyMethod("flush");
 
-    // Collect global errors
-    // Note: ApplicationInsightsAnalytics is the extension string identifier for
-    //  AppAnalytics. It is defined in ApplicationInsights.ts:ApplicationInsights.identifer
-    if (!(aiConfig.disableExceptionTracking === true ||
-        (aiConfig.extensionConfig &&
-        aiConfig.extensionConfig.ApplicationInsightsAnalytics &&
-        aiConfig.extensionConfig.ApplicationInsightsAnalytics.disableExceptionTracking === true))) {
+        // Collect global errors
+        // Note: ApplicationInsightsAnalytics is the extension string identifier for
+        //  AppAnalytics. It is defined in ApplicationInsights.ts:ApplicationInsights.identifer
+        if (!(aiConfig.disableExceptionTracking === true ||
```

Comment:
Also to get better compression
var extCfg = aiConfig.extensionConfig || {};
var aiAnalytics = extCfg.ApplicationInsightsAnalytics || {};
if (!(aiConfig.disableExceptionTracking === true || aiAnalytics.disableExceptionTracking === true)) {